react/utils/pages.ts

```diff
@@ -61,9 +63,13 @@ export function pathFromPageName(page: string, pages: Pages, params: any) {
     console.error(`Page ${page} has no path`)
     return null
   }
+  return adjustTemplate(template)
+}
 
-  const properTemplate = adjustTemplate(template)
-  return new RouteParser(properTemplate).reverse(params) || null
+export function pathFromPageName(page: string, pages: Pages, params: any) {
+  const validTemplate = getValidTemplate(page, pages)
+  if(!validTemplate) return null;
+  return new RouteParser(validTemplate).reverse(params) || null
```

Comment:
The `|| null` might be unnecessary here. If there's a truthy `validTemplate`, enforced in the line above, I think that there's no way the `reverse` will return falsy

Reply:
But this check is just for validTemplate, we still can have errors in params, then null could be necessary isn't?

Reply:
@luciannojunior `new RouteParser(validTemplate).reverse(params)` can return `false` as well.
This method was already with `|| null` before this changes.